chapter_09_docker.asciidoc

```diff
@@ -1064,9 +1095,11 @@ But we're just mapping 8888 to 8888, and that will look like this:
 
 [subs="specialcharacters,quotes"]
 ----
-$ *docker build -t superlists . && docker run -p 8888:8888 -it superlists*
+$ *docker run -p 8888:8888 -it superlists*
 ----
 
+TIP:  We don't need to run `docker build` this time since we have not changed our Dockerfile.
```

Comment:
I just found this. Now that we had a call with Harry recently that made it clear we were not mounting src/ in the next chapter on purpose, this is just partially true. 
Correctly: 

> "We don't need to run `docker build` this time since we are not changing our image."

The way I worded it originally suggests that we only need to rebuild when we change the Dockerfile itself (which is usually the case with most of my work) but since we use the COPY directive in the Dockerfile to bake the code in the image, modifying the source code also only takes effect after a rebuild is done.

I'll commit the change soon.